docs/getting-started/cli-getting-started.md

```diff
@@ -60,43 +60,58 @@ See [Understanding core command groups](../user-guide/cli-using-understanding-co
+Most command groups require a `zosmf-profile`, but some plug-ins add their own profile types. For example, the CICS plug-in has a `cics-profile`.
+
+The profile type that a command requires is defined in the `PROFILE OPTIONS` section of the `--help` response.
+
+:::tip
 
-**Tip:** The first `zosmf` profile that you create becomes your default profile. If you don't specify any options on a command, the default profile is used. Issue `zowe profiles -h` to learn about listing profiles and setting defaults.
+The first `zosmf` profile that you create becomes your default profile. If you do not specify any options on a command, the default profile is used. Issue `zowe config list --help` to learn about listing profiles and setting defaults.
 
-### Creating zosmf profiles
+:::
+
+### Creating z/OSMF profiles
+
+Create a global configuration file:
 
 ```
-zowe profiles create zosmf-profile myprofile123 --host my.company.com --port 123 --user myusername123 --password mypassword123
+zowe config init --global-config
 ```
 
-**Notes:** 
+A `zowe.config.json` file is saved in your home `~/.zowe` directory that includes a z/OSMF profile. Use a text editor to add or modify connection information in the profile.
+
+:::note Notes 
 
-- The port defaults to 443 if you omit the `--port` option. Specify a different port if your host system does not use port 443.
-- If z/OSMF is configured for high availability in Sysplex, create the CLI zosmf-profile with DVIPA address/hostname to ensure availability of REST services. For more information, see [Configuring z/OSMF high availability in Sysplex](../user-guide/systemrequirements-zosmf-ha.md).
+- Use the preceding command only when a configuration file does not already exist.
+- If z/OSMF is configured for high availability in Sysplex, create the CLI z/OSMF profile with a DVIPA address/hostname to ensure availability of REST services. For more information, see [Configuring z/OSMF high availability in Sysplex](../user-guide/systemrequirements-zosmf-ha.md).
 
-### Using zosmf profiles
+:::
+
+### Using z/OSMF profiles
 
 ```
-zowe zos-files download data-set "MY.DATA.SET(member)" -f "mylocalfile.txt" --zosmf-profile myprofile123
+zowe zos-files download data-set "MY.DATA.SET(member)" -file "mylocalfile.txt" --zosmf-profile myprofile123
 ```
 
-For detailed information about issuing commands, using profiles, and more, see [Using CLI](../user-guide/cli-using-usingcli.md).
+For detailed information about issuing commands, using profiles, and more, see [Using Zowe CLI](../user-guide/cli-using-usingcli.md).
 
 ## Writing scripts
 
 You can write Zowe CLI scripts to streamline your daily development processes or conduct mainframe actions from an off-platform automation tool such as Jenkins or TravisCI.
 
-### Example:
+### Example
+
+In this example scenario, you want to delete a list of temporary data sets.
```

Comment:
Just wondering (for the sake of consistency)...
Should we do something similar to the above examples?
`### Example: Downloading a partitioned data-set (PDS) member to local file`